gobblin-service/src/main/java/org/apache/gobblin/service/modules/flow/MultiHopFlowCompiler.java

```diff
@@ -66,6 +70,10 @@
 @Alpha
 @Slf4j
 public class MultiHopFlowCompiler extends BaseFlowToJobSpecCompiler {
+  private static final String DATASET_SUBPATHS_KEY = "dataset.subPaths";
```

Comment:
Should this property be moved to ConfigurationKeys and renamed as gobblin.flow.dataset.subPaths?